### DISCUSSION TITLE
Dropshipping plugins

### DISCUSSION THREAD

**Discussion by Archon-** [medusajs/medusa#7023] [Feature Requests]
I'm searching all over GitHub and I can't find any plugin integrating MedusaJS with dropshipping systems like CJDropshipping or AliExpress. I've seen a plugin for Printful, but that's the only thing I found. Has anyone seen anything like that?

I think it would be very useful because many sellers use the dropshipping model.

On a long run I'm happy to create plugin but just don't want to reinvent a wheel.

**Comment by MaxBrych** [comment]
Hey Archon.
Have you found or created anything yet?
